tests/queries/0_stateless/02483_add_engine_full_column_to_system_databases.sql

```diff
@@ -0,0 +1,5 @@
+DROP DATABASE IF EXISTS replicated_database_test; 
+CREATE DATABASE IF NOT EXISTS replicated_database_test ENGINE = Replicated('some/path/replicated_database_test', 'shard_1', 'replica_1');
```

Comment:
```suggestion
CREATE DATABASE IF NOT EXISTS replicated_database_test ENGINE = Replicated('some/path/' || currentDatabase() || '/replicated_database_test', 'shard_1', 'replica_1');
```

Reply:
Explanation: `clickhouse-test` generates unique database name for each test and it is used as current database for test. This change will guarantee that path is unique.